### ISSUE TITLE
[SUGGESTION] C# Dev Kit - Restore Solution Explorer project hierarchy

### ISSUE THREAD

**Issue by iseos** [microsoft/vscode-dotnettools#422]
### Describe the feature you'd like

Need an options to restore Solution Explorer project hierarchy state on solution load.
It is very inconvenient to open the project every time and expand the necessary folders.

### Environment Information

- OS: Windows 10
- VS Code: 1.81.1
- C# Dev Kit: 0.4.5

**Comment by odonyde** [comment]
I agree that this would improve the DX significantly. 👍🏻 Having to re-expand manually all worked on projects and folders in the Solution Explorer view after each restart of VS Code is very inconvenient.

As a limited work-around, I have activated the "Automatically Sync with Active Item" setting and I leave all files that I need for my current task open in the editor view. After VS Code restarted and the Solution Explorer view reloaded, I can click then one by one on each open editor tab and the corresponding items will be expanded and highlighted in the Solution Explorer view. Not great, but better than nothing.

**Comment by mindblower23** [comment]
Are there any news on this? This would be a really nice addition. And thanks, odonyde, for the work-around!
